roles/docker_socket_proxy/defaults/main.yml

```diff
@@ -0,0 +1,110 @@
+##########################################################################
+# Title:            Sandbox: docker_socket_proxy | Default Variables     #
+# Author(s):        salty, JigSawFr                                      #
+# URL:              https://github.com/saltyorg/Sandbox                  #
+# URL:              https://github.com/Tecnativa/docker-socket-proxy     #
+# --                                                                     #
+##########################################################################
+#                   GNU General Public License v3.0                      #
+##########################################################################
+---
+################################
+# Basics
+################################
+
+docker_socket_proxy_name: docker_socket_proxy
+
+################################
+# Docker
+################################
+
+# Container
+docker_socket_proxy_docker_container: "{{ docker_socket_proxy_name }}"
+
+# Image
+docker_socket_proxy_docker_image_pull: true
+docker_socket_proxy_docker_image_tag: "latest"
+docker_socket_proxy_docker_image: "tecnativa/docker-socket-proxy:{{ docker_socket_proxy_docker_image_tag }}"
+
+# Ports
+docker_socket_proxy_docker_ports_defaults: []
+docker_socket_proxy_docker_ports_custom: []
+docker_socket_proxy_docker_ports: "{{ docker_socket_proxy_docker_ports_defaults
+                                      + docker_socket_proxy_docker_ports_custom
+                                   if (not reverse_proxy_is_enabled)
+                                   else docker_socket_proxy_docker_ports_custom }}"
+
+# Envs
+docker_socket_proxy_docker_envs_default:
```

Comment:
We should give `CONTAINERS` and `IMAGES` as the current role does.

Unless the intention is to spin up a separate proxy for each app that may require it?

Reply:
depend, if we go to the way we are doing things like mariadb, postgres, it's a separate proxy for each role that needs this.

Reply:
> We should give `CONTAINERS` and `IMAGES` as the current role does.
> 
> Unless the intention is to spin up a separate proxy for each app that may require it?

Do you remember why you added images?

Reply:
Can't remember exactly - it may have been glances?

Reply:
> Can't remember exactly - it may have been glances?

Could you keep an eye on glances when I push this and the sandbox edit I have locally if we need to add that for glances?

Reply:
Found it - it was indeed a fix for glances

https://ptb.discord.com/channels/853755447970758686/1032051337376047204

Reply:
Super, I'll add it to my modification then.